ci_support/run_docker_build.sh

```diff
@@ -37,16 +39,11 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
+conda update --yes --all
+#conda install --yes conda-build==1.18.2
```

Comment:
Ideally we would always use latest, but we are being held back due to a bug. (I can't remember which one!)

For this package I need this https://github.com/conda/conda-build/pull/731